CustomSkillForDataIngestion/QnAIntegrationCustomSkill/QnAIntegrationCustomSkill.cs

```diff
@@ -83,7 +84,8 @@ public static class UploadToQnAMaker
                         {
                             Id = id,
                             FileName = blobName,
-                            FileUri = fileUri
+                            FileUri = fileUri,
+                            BlobPath = blobUrl.Remove(0, blobBaseURL.Length)
```

Comment:
why can't we do this on the file Uri won't that be a blob url with a SAS?

Reply:
sas token would expire right